index.html

```diff
@@ -1,122 +1,136 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>PictGame</title>
+	<title>Brain Game</title>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 
   <!-- SELECT CARDS TO PLAY -->
+
     <header id="main-header" class="bg-success text-white p-4 mb-3">
         <div class="container">
           <div class="row">
             <div class="col-md-6">
-                <h1 id="header-title">PictGame</h1>
+                <h1 id="header-title" style="text-align:center">Brain Game</h1>
             </div>
-            <div class="col-md-6 align-self-center">
-                <button id="play" class="play" ><img src="img/playButton.jpg" /></button>
+          </div>
+            <div class="col-md-6 align-self-center" style="text-align:center">
+                <button id="play" class="play" style="color: blue;">START</button>
                 <!-- <input type="button" value="PLAY" class="play" id="play"> -->
                 <select id="select-game"  class="select-game">
                     <option value="0">Please Select</option>
                     <option value="1">Play with 6 card</option>
                     <option value="2">Play with 12 card</option>
                     <option value="3">Play with 18 card</option>
                 </select>
-                <button id="reset" class="reset" ><img src="img/resetButton.jpg" /></button>
+                <button id="reset" class="reset" style="color: blue;">RESET</button>
             </div>
-          </div>
         </div>
       </header>
 
 
 <!-- PLAYING WITH 6 CARSD -->
 
-<div class="Tier1" id="Tier1">
-   <div class="row1">
-    <div id="box-1" class="box-1">
+<div class="Tier1" id="Tier1" >
+   <div class="row1" id="container1">
+    <div id="box-1" class="picBox">

```

Comment:
Los class name en HTML suelen ser en kebab-case [aqui puedes ver](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) las diferencias para que te familiarices

Reply:
lo cambie a kebab-case....estaba bueno el link